test/cmdlineargs.jl

```diff
@@ -61,15 +61,6 @@ let exename = `$(Base.julia_cmd()) --precompiled=yes --startup-file=no`
     @test !success(`$exename -L`)
     @test !success(`$exename --load`)
 
-    # --cpu-target
-    # NOTE: this test only holds true if image_file is a shared library.
```

Comment:
why don't we want to run this test still?

Reply:
> I'm not aware of any method to ask LLVM if it is a supported target name at runtime

You mean `jl_TargetMachine->getMCSubtargetInfo()->isCPUStringValid()` 😜 

(added in https://llvm.org/svn/llvm-project/llvm/trunk@223147)

Reply:
Doesn't seem to be a way to do that without looking up the CPU first?